packages/snage/src/config/migration/v0-v2.test.ts

```diff
@@ -0,0 +1,18 @@
+import {migrate} from '../migrate';
+import YAML from 'yaml';
+
+describe('v0 -> v2', () => {
```

Comment:
This error might appear in future migrations as well. What about making this test a bit more generic: Always migrate from `v0` to the current latest version?